apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/name-step/index.tsx

```diff
@@ -14,10 +14,9 @@ import { useTitle } from '@automattic/launch';
 import './styles.scss';
 
 const NameStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onNextStep } ) => {
-	const { title, updateTitle, saveTitle } = useTitle();
+	const { title, updateTitle } = useTitle();
 
 	const handleNext = () => {
-		saveTitle();
```

Comment:
In case (very remote) case the user types a title and then clicks the "next" button before 1s debounce ticks, would the debounced function still execute, and the title still be saved?

Reply:
After testing, we can confirm that the debounced function still executes even if the user presses the Next button